packages/contracts/deploy/helpers.ts

```diff
@@ -46,5 +53,80 @@ export async function updateActiveContractsJSON(payload: {
   );
 }
 
+export function ensLabelHash(label: string, ethers: any): string {
+  return ethers.utils.id(label);
+}
+
+export function ensDomainHash(name: string, ethers: any): string {
+  return ethers.utils.namehash(name);
+}
```

Comment:
Maybe we can reduce code duplication and collect them in a separate helper file, as they are also reused in the `ENSSubdomainRegistrar` tests.